src/makeRobot.js

```diff
@@ -39,6 +39,64 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(step = 1) {
+      if (step < 0) {
+        return this;
+      }
+      this.coords.y += step;
+
+      return this;
+    },
+
+    goBack(step = 1) {
+      if (step < 0) {
+        return this;
+      }
+      this.coords.y -= step;
+
+      return this;
+    },
+
+    goRight(step = 1) {
+      if (step < 0) {
+        return this;
+      }
+      this.coords.x += step;
+
+      return this;
```

Comment:
```suggestion
      if (step > 0) {
        this.coords.x += step;
      }

      return this;
```

make this simpler and not duplicate `return this`